pom.xml

```diff
@@ -73,6 +73,7 @@
         <dep.netty.version>4.1.92.Final</dep.netty.version>
         <dep.jna.version>5.13.0</dep.jna.version>
         <dep.okio.version>3.3.0</dep.okio.version>
+        <dep.failsafe.version>3.1.0</dep.failsafe.version>
```

Comment:
We should remove this and use `${dep.plugin.surefire.version}` instead, since Failsafe is part of Surefire and thus should share the same dependency version. It seems like the `surefire-testng` dependency below needs to match.